targets/TARGET_STM/TARGET_STM32L4/TARGET_STM32L496xG/TARGET_NUCLEO_L496ZG/system_stm32l4xx.c

```diff
@@ -0,0 +1,591 @@
+/**
```

Comment:
Please rebase. We used now a common file for all L4 devices. The clock config is no more done in it.